pages/glossary/programmatically-hidden.md

```diff
@@ -8,15 +8,29 @@ input_aspects:
   - DOM tree
 ---
 
-An HTML element is _programmatically hidden_ if either it has a [computed][] CSS property `visibility` whose value is not `visible`; or at least one of the following is true for any of its [inclusive ancestors][] in the [flat tree][]:
+An HTML element is _programmatically hidden_ if any of the following is true:
 
-- has a [computed][] CSS property `display` of `none`; or
-- has an `aria-hidden` attribute set to `true`
+- the element has a [computed][] CSS property `visibility` whose value is not `visible`; or
+- at least one [ancestor][] of the element has a [computed][] CSS property `content-visibility` of `hidden`; or
+- at least one [inclusive ancestors][] of the element has a [computed][] CSS property `display` of `none`; or
+- at least one [inclusive ancestors][] of the element has an `aria-hidden` attribute set to `true`
```

Comment:
Not new to the PR, we could open a separate issue, but this is incorrect I think. If you use aria-owns to grab a node that has an ancestor with aria-hidden, that node itself isn't hidden.

Reply:
Yes, that's also the conclusion that was reach [in this ARIA PR](https://github.com/w3c/aria/pull/1839#issuecomment-1341754741). 

I'm recording that in #1991 .